src/viam/sdk/components/camera.cpp

```diff
@@ -230,6 +230,7 @@ Camera::properties Camera::from_proto(
     const viam::component::camera::v1::GetPropertiesResponse& proto) {
     Camera::distortion_parameters distortion_parameters;
     Camera::intrinsic_parameters intrinsic_parameters;
+    Camera::mime_types mime_types;
```

Comment:
You didn't write this function this way, so if you don't fee like changing it I'll understand, but I'd expect the entire function body could be replaced with this:
```
return {
    proto.supports_pcd(),
    from_proto(proto.intrinsic_parameters()),
    from_proto(proto.distortion_parameters()),
    { proto.mime_types().begin(), proto.mime_types().end() }
};
```

See https://godbolt.org/z/oaPs318hf